entrypoint.sh

```diff
@@ -22,12 +22,12 @@ set -eou pipefail
 run theme-check --version
 
 if [[ -z "$INPUT_TOKEN" ]]; then
-  run theme-check $flags "$theme_root"
+  run theme-check $flags --update-docs "$theme_root"
```

Comment:
Given that folks can set the theme-check version (See $INPUT_VERSION), should we add a condition around this flag?